mulog-core/src/com/brunobonacci/mulog.clj

```diff
@@ -269,14 +270,34 @@ For more information, please visit: https://github.com/BrunoBonacci/mulog
   ```
 
   The local context can be nested and ti will be inherited by
-  all the **μ/log** calls within nested functions as long as they
+  all the ***μ/log*** calls within nested functions as long as they
   are in the same execution thread and which the scope of the block.
   "
   [context & body]
   `(binding [*local-context* (merge *local-context* ~context)]
      ~@body))
 
 
+(defmacro verbose [event-name & pairs]
+  `(with-context {:mulog/level ::lvl/verbose}
```

Comment:
what does this mean `::lvl/verbose` mean?? never seen this way of defining keywords.
is either `::keyword` or `:namespace/keyword`, but I never saw `::namespace/keyword` this before.

Reply:
are you sure it works?

```

user=> {:mulog/level ::lvl/fatal}
Syntax error reading source at (REPL:1:26).
Invalid token: ::lvl/fatal
```

Reply:
It's a feature of the reader to resolve auto-resolved keywords through their namespace alias, if present.

You can find a short example in the "official" destructuring guide's part related [to namespaced keywords][namespaced_keywords]:

> You can even destructure using auto-resolved keywords, which will again be bound to only the name part of the key:
>
> ```clojure
> ;; this assumes you have a person.clj namespace in your project
> ;; if not do the following at your repl instead: (create-ns 'person) (alias 'p 'person)
> (require '[person :as p])
>
> (let [person {::p/name "Franklin", ::p/age 25}
>       {:keys [::p/name ::p/age]} person]
>   (println name "is" age))
> 
> ;= Franklin is 25
> ```
> 
> Creating and destructuring maps with auto-resolved keywords allow us to write code using a namespace alias (here `p`) that is defined by a require in the current namespace, giving us a means of namespace indirection that can be changed at a single place in the code.

It's also mentioned briefly in the [reference doc][reference]'s part about reader literals, in the last bullet point (emphasis mine):

> * Keywords - Keywords are like symbols, except:
>   * They can and must begin with a colon, e.g. :fred.
>   * They cannot contain '.' in the name part, or name classes.
>   * Like symbols, they can contain a namespace, `:person/name`, which may contain '.'s.
>   * A keyword that begins with two colons is auto-resolved in the current namespace to a qualified keyword:
>       * **If the keyword is unqualified, the namespace will be the current namespace.** In user, `::rect` is read as `:user/rect`.
>       * **If the keyword is qualified, the namespace will be resolved using aliases in the current namespace.** In a namespace where x is aliased to example, `::x/foo` resolves to `:example/foo`.

I've noticed just now that in [your guide][brunobonacci_guide] you mention:

> double-colon `::` is a shortcut to represent current namespace

Which is true but omits the part on aliases. That's funny because I refer to your guide all the time and never noticed this! 😄  
I don't know if (or when) this has changed, but yes it actually works.

[namespaced_keywords]: https://clojure.org/guides/destructuring#_namespaced_keywords
[reference]: https://clojure.org/reference/reader#_literals
[brunobonacci_guide]: https://blog.brunobonacci.com/2014/11/16/clojure-complete-guide-to-destructuring/

Reply:
Thanks for the explanation, I didn't know that.
It makes total sense, having `::alias/keyword` pointing to a local alias and `::keyword` just to the local namespace.